### DISCUSSION TITLE
Button to remove all cache entries of a repo?

### DISCUSSION THREAD

**Discussion by mkurz** [community/community#36878] [Actions]
Hi!

It's great there is a user interface now to display a repo's caches and to remove particular cache entries: https://github.blog/changelog/2022-10-20-manage-caches-in-your-actions-workflows-from-web-interface/
Thanks for this!

However I miss a button to "remove all cache entries".... I want to clean the whole cache of a repository.
Is that on the roadmap? (Unfortunately I can not comment on https://github.com/github/roadmap/issues/502)

Thanks!

**Comment by bishal-pdMSFT** [comment]
@mkurz the inactive caches in a repository get auto cleaned up after 7 days (or earlier if repo's caches is above quota of 10GB storage). Can you please help with the use case for "delete all".

**Reply by covik** [reply]
I cache docker builds with `buildx` and  `cache-to=gha,mode=max`. While setting up the CI pipeline and the Dockerfile, things change rapidly so you end up with a lot of unused cache (which was recently created so it will take time until it gets auto cleaned up?). Also if a cache miss occurs when it should not, clearing all cache entries can help with keeping sanity.

**Reply by iTrooz** [reply]
it is useful when you are testing/debugging the caching capabilities of your workflow, and need to often delete a lot of entries.

I would change this feature request to "Delete multiple caches at the same time", and make an implementation similar to how we can take action on multiple issues/prs at the same time, with the checkbox next to them.
